net.meshlab.MeshLab.yaml

```diff
@@ -0,0 +1,98 @@
+app-id: net.meshlab.MeshLab
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15'
+command: meshlab
+rename-desktop-file: meshlab.desktop
+# Wayland does not work https://github.com/cnr-isti-vclab/meshlab/issues/738
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /share/doc
+  - /share/man
+  - /bin/rbox
+  - /bin/q*
+  - /bin/3dsdump
+  - "*.la"
+  - "*.a"
+
+modules:
+  - shared-modules/glew/glew.json
+  - shared-modules/glu/glu-9.json
+  
+  - name: Eigen3
+    buildsystem: cmake-ninja
+    builddir: true
+    cleanup: 
+      - "*"
+    sources:
+      - type: archive
+        url: https://gitlab.com/libeigen/eigen/-/archive/3.3.9/eigen-3.3.9.tar.bz2
+        sha256: 0fa5cafe78f66d2b501b43016858070d52ba47bd9b1016b0165a7b8e04675677
+        
+  - name: muparser
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://github.com/beltoforion/muparser/archive/refs/tags/v2.3.2.tar.gz
+        sha256: b35fc84e3667d432e3414c8667d5764dfa450ed24a99eeef7ee3f6647d44f301
+        
+  - name: lib3ds
+    buildsystem: autotools
+    sources:
+      - type: archive
```

Comment:
to fix the aarch64 build here try and add something like https://github.com/flathub/com.tux4kids.tuxmath/blob/adc11b752a572c744b2041ba43d65545af121207/com.tux4kids.tuxmath.json#L51